main.tf

```diff
@@ -80,6 +81,7 @@ resource "azurerm_kubernetes_cluster" "main" {
       type                         = var.agents_type
       vnet_subnet_id               = var.vnet_subnet_id
       zones                        = var.agents_availability_zones
+      ultra_ssd_enabled            = var.ultra_ssd_enabled
```

Comment:
Could we move this line to line 81 please? We're sorting these assignments by the following rules:

1. Head meta argument in the following order (`provider`, `count`, `for_each`)
2. Required arguments in alphabet order
3. Optional arguments in alphabet order
4. Required nested blocks in alphabet order
5. Optional nested blocks in alphabet order
6. Tail meta argument (`depends_on`)
7. Tail meta nested block(`lifecycle`)